src/git-repository-async.js

```diff
@@ -0,0 +1,272 @@
+'use babel'
+
+const Git = require('nodegit')
+const path = require('path')
+const {Emitter, Disposable, CompositeDisposable} = require('event-kit')
+
+// Temporary requires
+// ==================
+// GitUtils is temporarily used for ::relativize only, because I don't want
+// to port it just yet. TODO: remove
+const GitUtils = require('git-utils')
+// Just using this for _.isEqual and _.object, we should impl our own here
+const _ = require('underscore-plus')
+
+module.exports = class GitRepositoryAsync {
+  static open (path, options = {}) {
+    // QUESTION: Should this wrap Git.Repository and reject with a nicer message?
+    return new GitRepositoryAsync(path, options)
+  }
+
+  constructor (path, options) {
```

Comment:
Good catch - this currently conforms to [JS Standard Style](https://github.com/feross/standard) (a very presumptuous name, isn't it?) rather than AirBnB style and I'm happy with Standard.